src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -890,5 +890,82 @@ private static ErrorValue GetDiv0Error(IRContext irContext)
                 Message = "Division by zero"
             });
         }
+
+        private static FormulaValue Dec2Hex(IRContext irContext, NumberValue[] args)
+        {
+            var min_number = -Math.Pow(2, 39);
```

Comment:
>min_number [](http://example.com/codeflow?start=16&length=10)

`minNumber` - the coding standard used in this project is for variables to use camelCase. Same for other variables in this function.